### PR TITLE
feature/admin-can-import-rewards-from-CSV

### PR DESCRIPTION
**Pull request by rniedziolka**
Hi,
Admin can import rewards from CSV:
https://loom.com/share/79cbcdfc7b8044aa8a722ad570c0962f
Let me know if you have any suggestions.

### CODE REVIEW

File: app/services/import/reward_import.rb [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+require 'csv'
+
+module Import
+  class RewardImport
+    def initialize(data)
+      @data = data
+    end
+
+    def self.import(file)
+      ActiveRecord::Base.transaction do
```
Comment (oskarlakner) [inline app/services/import/reward_import.rb:12]: <3

File: config/routes.rb [hunk 2]
```diff
@@ -17,7 +17,9 @@
     resources :categories
     resources :category_rewards
     resources :company_values
-    resources :rewards
+    resources :rewards do
+      collection { post :import }
```
Comment (oskarlakner) [inline config/routes.rb]: this should be a block

File: spec/system/admin_reward_spec.rb [hunk 3]
```diff
@@ -46,7 +46,18 @@
...
+    attach_file(File.absolute_path('./spec/files/docs/example.csv'))
+    click_button 'Import CSV file'
+    expect(page).to have_content '86'
+    expect(page).to have_content 'czesc'
+    attach_file(File.absolute_path('./spec/files/docs/example2.csv'))
+    click_button 'Import CSV file'
+    expect(page).to have_content '58'
+    expect(page).to have_content 'bye'
+    expect(page).not_to have_content '86'
+    expect(page).not_to have_content 'czesc'
```
Comment (oskarlakner) [inline spec/system/admin_reward_spec.rb]: czesc

**Review by oskarlakner — APPROVED** [review]
well done champ!

**Comment by rniedziolka** [comment]
Thank you, mentor!

File: config/routes.rb (hunk 2 above)
Comment (rniedziolka) [reply config/routes.rb]: I don't quite understand what you mean.

Comment (rniedziolka) [reply config/routes.rb]: @oskarlakner - right?
```
resources :rewards do
  collection do
    post 'import'
  end
end
```

Comment (oskarlakner) [reply config/routes.rb]: yes

**Pull request merged**